README.md

```diff
@@ -11,6 +11,19 @@ computing.
 
 ![Jupyter notebook example](docs/resources/running_code_med.png "Jupyter notebook example")
 
+Since the release of JupyterLab, many users of the classic Jupyter Notebook have been migrating to using the new front-end.
+
+The `jupyter/notebook` repository is currently maintained by a skeleton crew of maintainers from the Jupyter community. Our approach is:
```

Comment:
Suggestion:
Our approach is: -> As a result, our approach moving forward will be: